src/TextInput.js

```diff
@@ -36,9 +36,9 @@ const TextInput = styled(TextInputBase)`
   box-shadow: ${get('shadows.formControl')};
 
   &:focus {
-    border-color: ${get('blue.4')};
+    border-color: ${get('colors.blue.4')};
```

Comment:
Was this broken before?

Reply:
yes, because I wasn't calling `colors` so it wasn't returning anything